app/js/TabWrangleOption.js

```diff
@@ -0,0 +1,71 @@
+/* @flow */
+
+import React from 'react';
+
+type Props = {
+  selectedOption?: string,
+  onChange?: () => void
+};
+
+const OPTIONS = [
+  { name: 'withDupes', text: 'With Duplicates' },
+  { name: 'exactURLMatch', text: 'Exact URL match' },
+  { name: 'hostnameAndTitleMatch', text: 'Hostname and Title match' },
+];
+export default class TabWrangleOption extends React.Component {
+  constructor(props: Props) {
+    super(props);
+
+    this.state = { selectedOption: this.props.selectedOption };
+    this.handleChange = this.handleChange.bind(this);
```

Comment:
Prefer property initializers instead of `bind` in the constructor:

```
handleChange = (e) => {
   ...
};
```

Reply:
Fixed this